### DISCUSSION TITLE
Free?

### DISCUSSION THREAD

**Discussion by ghandylan** [community/community#19081] [Copilot Conversations] [answered]
Will this be free for students? I checked my account and i cannot see Copilot in the benefits section.

**Comment by damoski2** [comment]
I believe Github Copilot will be free for everyone till sometimes august, thats when all users have to switch to the subscription based system for continual access to github copilot

**Comment by IP0** [comment]


**Reply by ffxsam** [reply]
Did you know that AI is actually rather pricey at scale? GitHub Copilot uses OpenAI under the hood, and OpenAI has [its own fees](https://openai.com/api/pricing/). GitHub has to price Copilot accordingly to cover those costs + a bit of a margin, because business. A one-time fee wouldn't make any sense in this case.

If you're a professional coder and can't afford $10/mo for a tool that greatly increases your productivity (I know it does mine), then.. 🤷‍♂️

**Comment by rafasha123** [comment]
https://github.com/github-community/community/blob/15daf29c96ff5fb997e13bed7a8d2a3201660f94/CODE_OF_CONDUCT.md

**Comment by secretgspot** [comment]
used all of us betas to train the ai on our own code and now greed sets in and put it behind a ridiculous pricing.   My trust in companies just dropped by multiple points.    Offer mediocre service (copilot) to train on my data/sweat and once it becomes somewhat usable force me to pay or else.. how is this any different than a drug dealer giving first done of crack for free just to hook them before forcing them into prostitution just to keep the habit

**Comment by melvincwng** [comment, marked as answer]
@ghandylan, based on Github's blog over [here](https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/), they mentioned that Github Copilot "...will also be free to use for verified students and maintainers of popular open-source projects."

Hope this clarifies.

Reference:
https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/

**Reply by melvincwng** [reply]
@ghandylan, if this reply is helpful, please kindly 'Mark as answer' so that the others are aware that this issue has already been addressed.

Thank you and have a great day ahead 😄

**Comment by MrPotatoXx** [comment]
Apparently if you have a student account you can still use github copilot for free.
